### DISCUSSION TITLE
UST stuck in Wormhole

### DISCUSSION THREAD

**Discussion by farmer8stone** [wormhole-foundation/wormhole#260] [Problems / bugs] [answered]
Hey, my UST is stuck in the wormhole. It says "awaiting completion on Solana".

TX id: 0x07f78e42b60f9da73feb4ee61baee3bb9c3d3b0352608729d3cc5551dbb89ec0
Solana wallet: 74NykFk38sn9Vr86us3zTNuJMm82FEVTpUUXrSDz4fgE

Thank you!

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/598PHT25w89kxFBE3VnyaPm5qReEn2XJHU69WMzp7ny5wiQansCwTPxiCWh5b9PpoMb8xLzVALuTwyi87W6eCohQ
